admin/app/views/workarea/admin/users/_cards.html.haml

```diff
@@ -35,6 +35,12 @@
                 = link_to model.created_by.name, user_path(model.created_by)
               - else
                 = t('workarea.admin.users.cards.not_applicable')
+            %li
+              %strong= t('workarea.admin.fields.last_logged_in_at')
+              = model.last_login
```

Comment:
This method already existed, but wasn't in use.

Reply:
That method is using `time_ago_in_words` in a view model, which I think that method is not defined in and probably throwing an error to cause the test in the build to fail.

Reply:
err, ok it includes the date helper.. so maybe its not that. Build is failing though

Reply:
should be resolved once https://github.com/workarea-commerce/workarea/pull/490 is merged.

Reply:
Yeah, that's the same failure that I was seeing locally.